cmd/gateway-common.go

```diff
@@ -389,6 +390,20 @@ func gatewayHandleEnvVars() {
 			logger.Fatal(err, "Unable to parse MINIO_GATEWAY_SSE value (`%s`)", gwsseVal)
 		}
 	}
+
+	if globalGatewayName == PANFSBackendGateway {
+		path := env.Get(config.EnvPanFSBucketPath, "")
+		if path == "" {
+			logger.Fatal(config.ErrMissingEnvPanFSBucketPath(nil),
+				"Unable to get default bucket path in PanFS")
+		}
+		if _, err := os.Stat(path); os.IsNotExist(err) {
+			logger.Fatal(config.ErrInvalidDefaultPanFSBucketPath(nil),
+				"Path is not valid")
+		}
+		globalPanFSDefaultBucketPath = path
```

Comment:
Why we define `globalPanFSDefaultBucketPath = ""` if we are failed here if `path == ""`? `globalPanFSDefaultBucketPath` is equal to  `path` alvays. Is it correct?

Reply:
If it just global variables let's name it like `globalPanFSBucketPath`

Reply:
The idea here is to set some default path to the global variable from the environment. Empty string is not accepted (empty string means that env is not set or set to the `""`) - so we reject them. Next, check for valid path and if it exists - set the global variable. So we always have valid path in that global variable

Reply:
OK, got it. I'm just confused by the **default** part in the variable name